addons/l10n_id_efaktur/models/account_move.py

```diff
@@ -137,6 +137,10 @@ def download_efaktur(self):
     def _generate_efaktur_invoice(self, delimiter):
         """Generate E-Faktur for customer invoice."""
         # Invoice of Customer
+        company_id = self.company_id
+        dp = company_id.currency_id.decimal_places
+        dp_product_id = self.env['ir.config_parameter'].sudo().get_param('sale.default_deposit_product_id')
```

Comment:
```suggestion
        dp = company_id.currency_id.decimal_places
```
You don't use them so no need to add them back